notebooks/lesson09-overfitting.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<ul><li>typo: "... address these ..."</li><li>should you add references? chapter 5 of the ds4biz book is pretty good</li><li>for homework you could get them to work through parts 2 &amp; 3. i don't think you will have enough time to cover everything in the lesson, so the polynomial part should prepare them well</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>